profiles/common/modules/features/multisite_translations/multisite_translations.module

```diff
@@ -312,29 +323,16 @@ function rules_action_mail_to_related_translation_role($subject, $message, $from
 
 
 /**
- * hook_menu
+ * Implements hook_form().
  */
- /*
-function multisite_translations_menu() {
-  $items['admin/config/regional/tmgmt/buildrole'] = array(
-    'title' => 'Build translation roles',
-    'description' => 'Settings related to translation roles',
-    'page callback' => 'drupal_get_form',
-    'page arguments' => array('multisite_translation_form'),
-    'access arguments' => array('administer tmgmt'),
-    'type' => MENU_LOCAL_TASK,
-  );
-
-  return $items;
-}
-*/
-
-
 function multisite_translation_form($form, &$form_state) {
   $form['submit'] = array('#type' => 'submit', '#value' => t('Rebuild translation roles'));
   return $form;
 }
 
+/**
+ * Called on form submittion.
+ */
```

Comment:
The pattern for documenting form submission handlers is "Form submission handler for xyz_form()."

If you need to refer to this in the future, there is also a nice example of this in the same section of the coding standards documentation: https://www.drupal.org/coding-standards/docs#forms